core/src/platform.h

```diff
@@ -33,3 +38,6 @@ std::string stringFromResource(const char* _path);
  * allocated file
  */ 
 unsigned char* bytesFromResource(const char* _path, unsigned int* _size);
+
+bool fetchData(std::unique_ptr<std::string> _url, std::stringstream& _rawData);
```

Comment:
I agree it's definitely "fetching data", I just think it can be more descriptive. "streamFromHttpSync" would be even better because now it's immediately clear that this is going over the network using http, getting a stream, and running synchronously